### PR TITLE
[PLAYER-4325] Update to API level 28

### PR DESCRIPTION
**Pull request by orobles-ooyala**
- Move to androidx
- Move all dependencies versions to project level

### CODE REVIEW

File: OoyalaSkinSampleApp/build.gradle [hunk 1]
```diff
@@ -1,18 +1,64 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
+    // Define versions in a single place
```
Comment (bychkovdmitry) [inline OoyalaSkinSampleApp/build.gradle:3]: What do you mean by "Define versions in a single place"? Are you going to move all these versions to build.gradle of the root project later? Or we can store all the versions in separate gradle file and apply them where needed:
apply from: 'gradle/versions.gradle'

Comment (orobles-ooyala) [reply OoyalaSkinSampleApp/build.gradle:3]: They are already defined in the root project for each app, as you can see below. This way when we work with core, all dependencies will have the same version and we will not have to manually force a desired version.

Comment (bychkovdmitry) [reply OoyalaSkinSampleApp/build.gradle:3]: I suggested moving versions to the root of all sample apps to avoid duplication.

**Comment by bychkovdmitry** [comment]
Agreed that further refactoring will be done here: https://jira.corp.ooyala.com/browse/PLAYER-4689

**Review by bychkovdmitry — APPROVED** [review]
LGTM, Oscar suggested to make further refactoring here: https://jira.corp.ooyala.com/browse/PLAYER-4689

**Comment by akshaydch11** [comment]
@orobles-ooyala Could you please fix merge conflicts? SDK PR is merged so we are good to merge this.

**Comment by orobles-ooyala** [comment]
Done @akshaydch11

**Comment by cacel** [comment]
LGTM

**Review by cacel — APPROVED** [review]

**Pull request merged**